linetools/isgm/abscomponent.py

```diff
@@ -445,8 +449,47 @@ def repr_vpfit(self, b=10.):
         s = '{} {:.5f} {:.5f} {:.2f} {:.2f} {:.2f} {:.2f}'.format(name,self.zcomp,0,b,0,logN,0)
         if len(self.comment)>0:
             s += '! {}'.format(self.comment)
+        s += '\n'
         return s
 
+    def repr_alis(self, T_kin=1e4,bturb=0.):
+        """
+        String representation for ALIS
+
+        Parameters
+        ----------
+        T_kin : float, optional
```

Comment:
Again, a Quantity makes more sense here.

I like using them as often as I can.  Can be
cumbersome but is likely to save you in the long run.